products/green_fast_track/models/intermediate/buffers/int_buffers__waterfront_access_pow.sql

```diff
@@ -2,11 +2,33 @@
 
 WITH pow AS (
     SELECT * FROM {{ ref("stg__waterfront_access_pow") }}
+),
+
+filtered_name_agency AS (
+    SELECT
+        variable_type,
+        name,
+        agency,
+        raw_geom
+    FROM pow
+    WHERE
+        upper(name) IN (
+            'BUSH TERMINAL PIERS PARK', 'SHERMAN CREEK STREET END PARKS'
+        )
+        OR upper(agency) != 'NYC DPR'
+),
```

Comment:
please help confirm this filter aligns with what we have in the spreadsheet for `Public Waterfront` variable